src/cryptsetup/cryptsetup-token-systemd-tpm2/cryptsetup-token-systemd-tpm2.c

```diff
@@ -0,0 +1,223 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <libcryptsetup.h>
+
+#include "cryptsetup-token.h"
+#include "hexdecoct.h"
+#include "memory-util.h"
+#include "tpm2-util.h"
+#include "luks2-tpm2.h"
+
+#define TOKEN_NAME "systemd-tpm2"
+#define TOKEN_VERSION_MAJOR "1"
+#define TOKEN_VERSION_MINOR "0"
+
+#define crypt_log_dbg(cd, x...) crypt_logf(cd, CRYPT_LOG_DEBUG, x)
+#define crypt_log_err(cd, x...) crypt_logf(cd, CRYPT_LOG_ERROR, x)
+#define crypt_log_std(cd, x...) crypt_logf(cd, CRYPT_LOG_NORMAL, x)
```

Comment:
hmm, not a fan of unnecessary abbreviations, and we usually call our log functions log_error(), log_info(), … or form derived from that. hence maybe:

```c
crypt_log_dbg() → crypt_log_debug()
crypt_log_err() → crypt_log_error()
crypt_log_std() → crypt_log()
```

or so? doesn't matter much though